src/clouds.cpp

```diff
@@ -58,13 +57,13 @@ Clouds::Clouds(scene::ISceneManager* mgr,
 	//m_material.MaterialType = video::EMT_TRANSPARENT_VERTEX_ALPHA;
 	m_material.MaterialType = video::EMT_TRANSPARENT_ALPHA_CHANNEL;
 
+	m_params.height        = cloudheight ? cloudheight : 120;
```

Comment:
It's not much of a change, 120 would make the clouds look like they do from sea level in-game, which seems fine.